directory/cli/src/user.py

```diff
@@ -400,3 +490,14 @@ def _run_post_create_script(login):
         except subprocess.CalledProcessError as ex:
             error = script_result.stdout if error_in_stdout else script_result.stderr
             raise IpaRunError(error) from ex
+
+def _get_group_id(group):
+    try:
+        return list_command.find_data(
+                'group-find',
+                [group],
+                all_fields=False
+                )[0].get('GID')
+    except IpaRunError:
+        error = '{}: group not found'.format('clusterusers')
```

Comment:
There's nothing specific to `clusterusers` in this function (even if it is only ever called with that argument atm), so always formatting this string with `'clusterusers'` is likely to lead to a confusing error from this line in future - I think it would be better to use the passed in `group` 
```suggestion
        error = '{}: group not found'.format(group)
```